viewing_party/main.py

```diff
@@ -0,0 +1,161 @@
+def create_movie(movie_title, genre, rating):
+
+    # Create a new movie
+    new_movie = {"title": movie_title, "genre": genre, "rating": rating}
+
+    if new_movie["title"] == None or new_movie["genre"] == None or new_movie["rating"] == None:
+        return None
+    return new_movie
+
+def add_to_watched(user_data, movie):
+
+    # Add movie in the list of movies that the user has already watched
+    user_data = {
+        "watched": [movie]
+    }
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+
+    # Add movie in the user's watchlist
+    user_data = {
+        "watchlist": [movie]
+    }
+    return user_data
+
+def watch_movie(user_data, title):
+
+    # Check if title is in a movie in the user's watchlist 
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+        # Add movie in the list of watched movie and remove it from the watchlist
+            user_data["watched"].append(movie)
+            user_data["watchlist"].remove(movie)
+            return user_data
+    return user_data
+
+def get_watched_avg_rating(user_data):
+
+    # Setup
+    movie_rating = []
+    avg_rating = None
+
+    # Check if there are movies in the list of watch movies
+    if len(user_data["watched"]) == 0:
+        avg_rating = 0.0
+        return avg_rating 
+    # Check if there are any movie in the list of watch movies
+    else:
+        for movie in user_data["watched"]:
+        # Add movie in a new list
+            movie_rating.append(movie["rating"])
+        # Calculate the average rating
+        avg_rating = sum(movie_rating)/len(movie_rating)
+        return avg_rating
```

Comment:
this works great! but let's try and use only one `return` statement. also, the function `sum()` already runs a for-loop behind the scenes, so below we have turned two for-loops into only one!

```suggestion
    sum = 0
    avg_rating = 0.0

    # Check if there are movies in the list of watch movies
    if len(user_data["watched"]) > 0:
        for movie in user_data["watched"]:
            sum += movie["rating"]
        avg_rating = sum/len(movie_rating)
        
    return avg_rating
```